src/main/java/com/scalar/db/storage/cosmos/ResultImpl.java

```diff
@@ -148,7 +148,7 @@ private Value convert(Object recordValue, String name, String type)
       case "varchar":
         return new TextValue(name, (String) recordValue);
       case "blob":
-        return new BlobValue(name, (byte[]) recordValue);
+        return new BlobValue(name, ((String) recordValue).getBytes());
```

Comment:
I feel it should specify an encoding to make it able to work with non-ascii.
```suggestion
        return new BlobValue(name, ((String) recordValue).getBytes(StandardCharsets.UTF_8));
```

Reply:
Thank you! I've fixed it.